<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dualshock 4 tools</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>169</width>
      <height>207</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_RGB">
     <item>
      <widget class="QLabel" name="HeaderRGB">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>RGB</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_RGB_lables">
       <item>
        <widget class="QLabel" name="label_red">
         <property name="text">
          <string>Red: 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_green">
         <property name="text">
          <string>Green: 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_blue">
         <property name="text">
          <string>Blue: 0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_Dials">
       <item>
        <widget class="QDial" name="dial_red">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="wrapping">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="dial_green">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="dial_blue">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGraphicsView" name="RGBview"/>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_automaticRGB">
       <property name="text">
        <string>AutoChange</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
